scripts/mbedtls_dev/bignum_common.py

```diff
@@ -263,6 +270,14 @@ def from_montgomery(self, val: int) -> int:
     def boundary(self) -> int:
         return self.int_n
 
+    @property
+    def arg_a(self) -> str:
+        if self.mongtomgery_form_a:
```

Comment:
Here and elsewhere, spelling:
```suggestion
        if self.montgomery_form_a:
```

Reply:
bah - nice spot!